_includes/download_details_pc.md

```diff
@@ -1,3 +1,7 @@
+In addition to the plain command line image, there is a graphical flavor with the XFCE desktop environment.
+Other graphical environments are fully supported by Void Linux, but are not offered as installation images
+to avoid increasing the testing effort.
```

Comment:
`but are not offered as demonstration/installation images, to decrease the overhead involved with testing releases.` ?

Reply:
Done, thanks!